.github/workflows/ci.yml

```diff
@@ -411,6 +417,9 @@ jobs:
     env:
       MODIN_ENGINE: ${{matrix.engine}}
       MODIN_MEMORY: 1000000000
+      MKL_NUM_THREADS: 1
```

Comment:
@anmyachev, which threading technologies are hidden by these env vars except openmp?

Reply:
I don't know what technologies they use internally, but even if they are all based on the same technology, isn't it better to explicitly specify different environment variables rather than hope that `OMP_NUM_THREADS` will disable parallelization everywhere? [Dask](https://docs.dask.org/en/latest/array-best-practices.html#avoid-oversubscribing-threads) recommends doing just that. (OPENBLAS_NUM_THREADS should be added too)

Reply:
I think If all the packages use the same threading technology, there is no point to export different env vars. However, that is true if we exactly know there are no other threading technologies under the hood.